src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -480,30 +504,58 @@ export default class RadarTargetModel {
     }
 
     /**
-     * Set the value of the scratchpad
+     * Clear any existing halo
      *
      * @for RadarTargetModel
-     * @method setScratchpad
-     * @param scratchPadText {string}
+     * @method removeHalo
      * @return {array} [success of operation, system's response]
      */
-    setScratchpad(scratchPadText) {
-        this.scratchPadText = scratchPadText;
+    removeHalo() {
+        if (!this.hasHalo) {
+            return;
+        }
 
-        return [true, 'SET SCRATCHPAD'];
+        this._haloRadius = INVALID_NUMBER;
+
+        return [true, 'TOGGLE HALO'];
     }
 
     /**
-     * Toggle halo (circle) on and off
+     * Set the radius of the halo
      *
      * @for RadarTargetModel
-     * @method toggleHalo
+     * @method setHalo
+     * @param radius {number}
      * @return {array} [success of operation, system's response]
      */
-    toggleHalo() {
-        this._hasHalo = !this._hasHalo;
+    setHalo(radius) {
```

Comment:
I changed the name to `setHalo` to better reflect what the method is doing and to be in line with `removeHalo`. Maybe `toggleHalo` was already the best name.

Reply:
Eh, that'll work. Better than toggle either way, because it now will adjust the halo size if you do `.setHalo(4); .setHalo(7);`, so it doesn't always "toggle" it on and off each call anymore.

That should be fine :+1: